internal/zinc-persist/src/test/scala/sbt/inc/AnalysisGenerators.scala

```diff
@@ -170,7 +170,9 @@ trait AnalysisGenerators {
       external <- someOf(src.external.all.toList)
     } yield Relations.makeClassDependencies(Relation.empty ++ internal, Relation.empty ++ external)
 
-  def genScalaName: Gen[String] = Gen.listOf(Gen.choose('!', 'Z')).map(_.toString())
+  def genScalaName: Gen[String] = {
+    Gen.listOf(Gen.oneOf(Gen.choose('!', 'Z'), Gen.const('\n'))).map(_.toString())
```

Comment:
Did you try this before and after the change to make sure that it reproduced the bug?

Reply:
@jvican Yes